WikibaseDataModel.php

```diff
@@ -7,7 +7,10 @@
  * @author Jeroen De Dauw < jeroendedauw@gmail.com >
  */
 
-define( 'WIKIBASE_DATAMODEL_VERSION', '5.0.0 alpha' );
+/**
+ * @deprecated since 5.0
```

Comment:
See https://github.com/wmde/WikibaseDataModel/blob/4.4.0/WikibaseDataModel.mw.php#L11 or the discussion in https://gerrit.wikimedia.org/r/268401.